content/ways-to-train.md

```diff
@@ -110,104 +120,128 @@ calls_to_action:
       - "Assessment only"
 ---
 
-To teach in a state school in England, you must have a degree, and gain Qualified Teacher Status (QTS). There are a number of ways to gain QTS, such as completion of postgraduate initial teacher training (ITT) or an undergraduate degree course with QTS included.
+You need to get Qualified Teacher Status (QTS) to be a primary or secondary teacher in a state school in England.
 
-### Things to consider
+You can get QTS in different ways.
```

Comment:
Might it be worth listing those ways here and linking to the relevant section below?

Reply:
that's a good idea, although I'm about to do a more structural review of this page so maybe we can park that one?